.goreleaser.yaml

```diff
@@ -16,6 +16,7 @@ builds:
   - goos: linux
     goarch: arm64
   ldflags:
+  - -X github.com/bitrise-io/bitrise/version.VERSION={{ .Version }}
```

Comment:
Shouldn't `github.com/bitrise-io/bitrise/version.VERSION=` be `github.com/bitrise-io/stepman/version.VERSION=`

Reply:
Fixed.